pkgs/by-name/ha/handheld-daemon/package.nix

```diff
@@ -26,6 +27,7 @@ python3.pkgs.buildPythonApplication rec {
     rich
     setuptools
     toybox
+    xlib
```

Comment:
Why xlib?

Reply:
Xlib seems to be a dependency now:
![image](https://github.com/NixOS/nixpkgs/assets/39011842/0b1d136f-41d0-4d5e-aca4-3c9ebdbc6ff7)